dpgen/ops/dispatcherop.py

```diff
@@ -0,0 +1,118 @@
+from abc import abstractmethod
+from pathlib import Path
+from typing import List, Set
+from itertools import chain
+
+from dpdispatcher import Machine, Resources, Task, Submission
+
+from .op import OP, Status
+from .opio import OPIO
+from .context import IterationContext
+
+
+class DispatcherOP(OP):
+    r"""This OP will call DPDispatcher to execuate commands on remote.
+    
+    Parameters
+    ----------
+    context : IterationContext
+        DP-GEN iteration context
+    machine : dict
+        DPDispatcher machine parameters. See also DPDispatcher doc.
+    resources : dict
+        DPDispatcher machine parameters. See also DPDispatcher doc.
+    command : str
+        path or command to a certain program on remote
+    """
+    @OP.set_status(status = Status.INITED)
+    def __init__(
+            self,
+            context: IterationContext,
+            machine: dict,
+            resources: dict,
```

Comment:
How about input Machine and Resources objects?

Reply:
That's okay, but we also need to convert dict to object outside the OP